Makefile

```diff
@@ -32,6 +32,7 @@ crossbuild: web
 	mkdir -p $(BUILD_DIR)/$(BIN_DIR)
 	${MAKE} build
 	cp config.mainnet.json $(BUILD_DIR)/$(BIN_DIR)/default.json
+	cp -r certs $(BUILD_DIR)/$(BIN_DIR)/certs
```

Comment:
Let's use `cp -a` to be consistent with other cp.
Also, could you please add `rm -rf $(BUILD_DIR)/$(BIN_DIR)` right after `crossbuild: web` line? Otherwise it will cause certs and web directory to duplicate twice if we type `make all` twice... I just noticed this is a current bug

Reply:
done